plugins/modules/snap_alias.py

```diff
@@ -106,25 +105,26 @@ class SnapAlias(CmdStateModuleHelper):
         ],
         supports_check_mode=True,
     )
-    command = "snap"
-    command_args_formats = dict(
-        _alias=dict(fmt=lambda v: [v]),
-        state=dict(fmt=lambda v: [_state_map[v]]),
-    )
-    check_rc = False
+
+    command_args_formats = {
+        "state": cmd_runner_fmt.as_map(_state_map),
+        "name": cmd_runner_fmt.as_list(),
+        "alias": cmd_runner_fmt.as_list(),
+    }
 
     def _aliases(self):
         n = self.vars.name
         return {n: self._get_aliases_for(n)} if n else self._get_aliases()
 
     def __init_module__(self):
         self.vars.set("snap_aliases", self._aliases(), change=True, diff=True)
```

Comment:
I think this line indirectly needs `self.runner` to be present.

Reply:
done